### DISCUSSION TITLE
Multi stage stream

### DISCUSSION THREAD

**Discussion by rohrit** [airbytehq/airbyte#46711] [Low-code Framework] [answered]
Hi,

Is it possible to author a stream where the POST request uses output of a previous GET request as input in the body? I was thinking that the parent stream concept could be used here but it looked like the output from the parent stream can be used as query parameters but not in the body?

I am new to Airbyte and probably missed something in the documentation that allows this. Please let me know how this can be accomplished.

Thanks!

**Comment by natikgadzhi** [comment, marked as answer]
Yes! You probably want to make your initial GET request a "parent" stream, and then use that as a parent in the second stream that will be the "child" that can use the results of the first stream in it's request body.

**Reply by rohrit** [reply]
Hi @natikgadzhi , Thank you for your response. I will try it out and update here.
